cmd/ctraffic/main.go

```diff
@@ -360,18 +361,22 @@ func (c *config) clientMain() int {
 
 	wg.Wait()
 
+	c.printStats(s)
+	return 0
+}
+
+func (c *config) printStats(s *statistics) {
 	if *c.stats != "none" {
 		c.copyStats(s)
 		s.reportStats()
 	}
-
-	return 0
 }
 
 func (c *config) copyStats(s *statistics) {
+	var i uint32
 	if *c.stats == "all" {
 		s.ConnStats = make([]connstats, nConn)
-		for i := range s.ConnStats {
+		for i = 0; i < nConn && uint32(len(cData)) < i; i++ {
```

Comment:
What is the role of this guard equation?
`uint32(len(cData)) < i`
Can this expression became true? I suppose len(cData) is always greater than 0 and i starts from 0.